website-frontend/tests/layout.spec.ts

```diff
@@ -0,0 +1,70 @@
+import { test, expect } from '@playwright/test';
+
+const EXPECTED_SITE_TITLE = 'UP Diliman - Department of Computer Science';
+
+test.describe('2-4: Navigation Bar Links to Home', () => {
+	test('Check for home button in navbar', async ({ page }) => {
+		await page.goto('/');
+		const nav = await page.getByRole('navigation');
+		const home = await nav.getByRole('link', { name: 'Home' });
+		await home.click();
+		await page.waitForTimeout(5000);
+		expect(await page.title()).toBe(EXPECTED_SITE_TITLE);
+	});
+});
```

Comment:
Instead of defining `EXPECTED_SITE_TITLE`, I think we can use the REST API to obtain `global.title` (similar on how we obtain the collections and other CMS data using `curl`). This is to ensure that despite changes to the website title in the CMS, we will still get the expected results in the site navigation.

We can inherit the behavior of `collections.spec.ts` file and combine the page data to create a test that utilizes both `page` and `request` to test the correctness of navigation.

The following specifies my intended feedback on the current code structure. Kindly double check if this works, and also, feel free to modify this code for production:
```ts
test('page title test', async ({ page, request }) => {
  const res = await request.get(`${process.env.PUBLIC_APIURL}/items/global`);
  const global_title = await res.json().then((obj) => { return obj.data.title; })
  expect(await page.title()).toBe(global_title);
}
```

I am not sure how to implement this using the Directus SDK (e.g., `getDirectusInstance` in SvelteKit). Right now though, the straightforward way to test that the title matches what is specified in the CMS is through cross-referencing what is in the CMS and what is displayed in the website.

Reply:
Hi! My latest changes are to comply with this request, please refer to them, thanks!